pom.xml

```diff
@@ -295,6 +296,14 @@
           </archive>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>com.github.spotbugs</groupId>
+        <artifactId>spotbugs-maven-plugin</artifactId>
+        <configuration>
+          <xmlOutput>true</xmlOutput>
+          <failOnError>false</failOnError>
```

Comment:
It's better enabled. We can add exculde rules to avoid changing too much in the first place.

Reply:
@liubao68 you are right, I will fix it this weekend.